frontend/src/App.css

```
@@ -0,0 +1,376 @@
+/* ── Global Reset ──────────────────────────────────────────── */
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700;800;900&display=swap');
```

Stylelint (17.6.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700;800;900&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700;800;900&display=swap'" (import-notation)

(import-notation)